python/ray/autoscaler/_private/event_system.py

```diff
@@ -62,19 +135,22 @@ def add_callback_handler(
                 registered against.
             callback (Callable[[Dict], None]): Callable object that is invoked
                 when specified event occurs.
+            *args: Variable length arguments to be injected into callbacks.
+            **kwargs: Keyword arguments to be injected into callbacks.
         """
-        if event not in CreateClusterEvent.__members__.values():
+        if event not in event_enum_values:
             cli_logger.warning(
                 f"{event} is not currently tracked, and this"
                 " callback will not be invoked."
             )
```

Comment:
It doesn't look like this warning is accurate. Maybe we could either change the wording, or just remove it.